tools/generators/release-package/README.md

```diff
@@ -0,0 +1,45 @@
+# release-package
```

Comment:
not 100% sure about the name , my initial idea was to name it `prepare-release-package` as this wont release anything per say. WDYT @ling1726

Reply:
or maybe `prepare-package-release`